README.md

```diff
@@ -63,16 +63,42 @@ That's it!
   	```
 3. Insert the following lines inside the dependencies block in `android/app/build.gradle`:
   	```
-    compile project(':rn-salesforce-chat')
+    implementation project(path: ':@loadsmart_rn-salesforce-chat')
   	```
 
-4. Open `android/app/src/main/java/[...]/MainActivity.java`
+4. Open `android/app/src/main/java/[...]/MainApplication.java`
   - Add `import com.rn.salesforce.chat.RNSalesforceChatPackage;` to the imports at the top of the file
   - Add `new RNSalesforceChatPackage()` to the list returned by the `getPackages()` method    
 
+## Prepare App Submission for iOS
+
+Before you can submit your app to the App Store, you need to strip development resources (such as unneeded architectures and header resources) from the dynamic libraries used by the Salesforce Service SDK.
+
+Xcode doesn't automatically strip unneeded architectures from dynamic libraries, nor remove some header and utility resources. Apps that don't do this clean up are rejected from the App Store. For example, you might receive the following error:
+
+```
+ERROR ITMS-90085:
+No architectures in the binary. Lipo failed to detect any architectures in the bundle executable.
+```
+
+You can resolve this problem by using the script provided in the Salesforce ServiceCore framework that automatically strips unneeded architectures from the dynamic libraries and then re-signs them. To use this script:
+
+1. Select Build Phases for your project target.
+2. Create a Run Script phase to run the script.
+3. Copy the script bellow and paste on the script box:
+```sh
+if [ "${CONFIGURATION}" = "Release" ]; then
+    $PODS_ROOT/ServiceSDK/Frameworks/ServiceCore.xcframework/ios-arm64/ServiceCore.framework/prepare-framework
+else
+    $PODS_ROOT/ServiceSDK/Frameworks/ServiceCore.xcframework/ios-x86_64-simulator/ServiceCore.framework/prepare-framework
+fi;
+```
+
+This build phase **must occur after the link phase and all embed phases**. If you're using CocoaPods, make sure to put this script after the "[CP] Embed Pods Frameworks" phase.
+
```

Comment:
❓ Should this be done every single time we upload a version to the testflight?

Reply:
Yes, we need to do this every time.